rudder-web/src/main/webapp/javascript/rudder/homePage.js

```diff
@@ -234,10 +234,20 @@ function doughnutChart (id,data,count,colors) {
                         , value     : data[0]._model.label
                        }];
                       break;
+                  case 'nodeCompliance':
+                     var compliance = {
+                        "Poor"   : {min: 0,   max: 49}
```

Comment:
max should be 50 !!  (compliance is a float so it need to to 49.99999.... )

Reply:
Same for all other value, Also perfect should only have a min value (see my other comment)